src/makieplotting.jl

```diff
@@ -307,6 +307,95 @@ function Makie.plot!(AR::Arrows{<:Tuple{<:MakiePlotter{dim}}}) where dim
     Makie.arrows!(AR, ps, ns, arrowsize=AR[:arrowsize], normalize=AR[:normalize], colormap=AR[:colormap], color=lengths, lengthscale=AR[:lengthscale])
 end
 
+"""
+    elementinfo(ip::Interpolation; kwargs...)
+    elementinfo(cell::AbstractCell; kwargs...)
+    elementinfo(ip::Type{Interpolation}; kwargs...)
+    elementinfo(cell::Type{AbstractCell}; kwargs...)
+
+- `plotnodes::Bool=true` plots the nodes as circles/spheres
+- `strokewidth::Int=2` how thick faces/edges are drawn
+- `color::Symbol=theme(scene,:linecolor)` color of the faces/edges and nodes
+- `markersize::Int=30` size of the nodes
+- `nodelables=false` global node id labels
+- `nodelabelcolor=:darkblue`
+- `textsize::Int=15` size of the label's text
+"""
+@recipe(Elementinfo) do scene
+    Attributes(
+    plotnodes=true,
+    strokewidth=2,
+    color=theme(scene, :linecolor),
+    markersize=30,
+    textsize=60,
+    nodelabels=true,
+    nodelabelcolor=:darkred,
+    nodelabeloffset=(0.0,0.0),
+    facelabels=true,
+    facelabelcolor=:darkgreen,
+    facelabeloffset=(0,0),
+    edgelabels=true,
+    edgelabelcolor=:darkblue,
+    edgelabeloffset=(-40,-40),
+    )
+end
+
+function Makie.plot!(Ele::Elementinfo{<:Tuple{<:Ferrite.Interpolation{dim,refshape}}}) where {dim,refshape}
```

Comment:
We should leave some kind of todo here for the case when we add non-nodal interpolations like e.g. Nedelec. Maybe it even makes sense to just add a argument, i.e. i-th dof, to just plot one basis function. This way we can circumvent the direct association to a node. What do you think?

Reply:
yes we can definitely change things within the `Makie.plot!` dispatch, but it's no problem, since we anyways only expose `elementinfo` to the user, if there will be an additional `kwarg` then the interface doesn't really change. I think even for non-nodal interpolations we can just have those dispatches with the same interface (except for additional `kwargs`)